ubuntu1604-desktop.json

```diff
@@ -4,10 +4,10 @@
   "desktop": "true",
   "cpus": "1",
   "disk_size": "130048",
-  "iso_checksum": "70db69379816b91eb01559212ae474a36ecec9ef",
+  "iso_checksum": "805337c2c3a00ac9b4a59a5c9692903ad30fe3ce",
   "iso_checksum_type": "sha1",
-  "iso_name": "ubuntu-16.04-server-amd64.iso",
```

Comment:
@oinume , does `ubuntu-16.04.1-desktop-amd64.iso` really work for you?
There has been an automation(boot_command) problem with the desktop iso, so server iso was used instead(ubuntu-desktop is installed via provision scripts later). 
More discussion: https://github.com/boxcutter/ubuntu/issues/63